OWNERS_ALIASES

```diff
@@ -298,10 +287,7 @@ aliases:
   # sig-apps-emeritus:
   # - tnozicka
 
-  sig-autoscaling-maintainers:
-    - bskiba
-    - MaciekPytel
-    - mwielgus
```

Comment:
hmm... seems like there should at least be someone from https://github.com/kubernetes/community/tree/master/sig-autoscaling here... @mwielgus, can you weigh in or provide an alternate reviewer?

Reply:
Adding them back for now. haven't heard from them so far.